keyboards/handwired/prkl30/keymaps/erkhal/readme.md

```diff
@@ -0,0 +1,16 @@
+# The default keymap for PRKL30
+
+Layout geared towards productivity with OSX
+-----
+
+Default layer
+![prkl30 default layout image](https://i.imgur.com/tekFaHW.png "default")
+
+Lower
+![prkl30 lower layout image](https://i.imgur.com/5F4CL4g.png "lower")
+
+Raise
+![prkl30 raise layout image](https://i.imgur.com/BpC8T6p.png "raise")
+
+FN
+![prkl30 fn layout image](https://i.imgur.com/gwgIKqj.png "fn")
```

Comment:
Just a bit of visual polish. Space-Space-NewLine sequences in Markdown get turned into `<br>` elements in HTML.


```suggestion
Default layer  
![prkl30 default layout image](https://i.imgur.com/tekFaHW.png "default")

Lower  
![prkl30 lower layout image](https://i.imgur.com/5F4CL4g.png "lower")

Raise  
![prkl30 raise layout image](https://i.imgur.com/BpC8T6p.png "raise")

FN  
![prkl30 fn layout image](https://i.imgur.com/gwgIKqj.png "fn")
```